### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by zairdon20**
### In this project I did the following:
- Refactored the Calculator component (components/Calculator.js) from a class based component to a functional component with hooks.

### CODE REVIEW

**Review by ZeenatLawal — APPROVED** [review]
Hi @zairdon20, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by BertrandConxy — APPROVED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages,
:heavy_check_mark: Descriptive Readme file,
:heavy_check_mark: Great Job with Gitflow.
:heavy_check_mark: All Linters are passing
:heavy_check_mark: Great implementation of hooks
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,69 @@
...
+- run the project locally by entering the command npm start
+
+
+
+## Project Status
+This project is in progress.
+
+## Screenshoots
+![screenshot](src\assets\calculatorpic.png)
+## Live Demo (if available)
```
Comment (BertrandConxy) [inline README.md:39]: [OPTIONAL] It will be better if you deploy your app and include the link here

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+
+class App extends React.Component {
+  constructor(props) {
```
Comment (BertrandConxy) [inline src/App.js:5]: [OPTIONAL] It will be better if you refactor this App.js component to be function-based too

**Review by BertrandConxy — COMMENTED** [review]
Keep it up @zairdon20

**Pull request merged**